go/pkg/filemetadata/filemetadata.go

```diff
@@ -79,6 +106,18 @@ func Compute(filename string) *Metadata {
 		md.IsDirectory = true
 		return md
 	}
+
+	if len(XattrDigestName) > 0 && XattrAccess.isSupported() {
+		xattrValue, err := XattrAccess.getXAttr(filename, XattrDigestName)
+		if err == nil {
```

Comment:
My intention was that if there's an error (including "attribute not found" and "not supported") it would fall back on the old file hashing. Also didn't want to return the error because getXAttr returns an errnoErr but the Compute() comments promise it'll return a FileError and I'm worried of the possibility that someone has made that assumption somewhere and suddenly returning something else will cause problems, so I'd have to convert.
I could just log the error before falling through?

Reply:
I think we do want to return an error here - and converting is ok. 
Presumably the user will intentionally turn request the configuration/feature at a build level, and it'd generally be frustrating for it to just silently error.

Reply:
+1 to _not_ failing silently and bubbling up the error!

Reply:
Fair enough.  Fixed, and I've also made it error when xattr is not supported.